package.json

```diff
@@ -89,5 +89,8 @@
     "rimraf": "^2.6.2",
     "rsvp": "^4.8.4",
     "typescript": "~3.4.3"
+  },
+  "dependencies": {
+    "yarn": "^1.15.2"
```

Comment:
Looks like you've accidentally added yarn as a package dependency. Can you revert the package.json and yarn.lock change?

Reply:
confirm, we do not want to change yarn deps here

Reply:
Oh, yes. that's right. I will remove these changes.